app/controllers/games_controller.rb

```diff
@@ -1,2 +1,20 @@
 class GamesController < ApplicationController
+
+  def new
+    @game = Game.new
+  end
+
+  def create
+    @game = Game.new(game_params)
+    # @game.user_id = current_user.id
```

Comment:
This should be 
`@game.user = current_user`

Reply:
Hi Soo, I left i commented that out because I thought we needed to add that after we introduced pundit.

Reply:
I think you still need it, beause @game will need to be assigned to a user when it's created (as it has a user_id), which is the current user.
Hence the
`@game.user = current_user`

You don't need it to be user_id as you're calling the instance of the user for whom the game belongs to

Reply:
Okay understood, I'll update that, thanks for the check. I'll also look at yours when I get the chance.